test/fixtures/cloudiot/main.tf

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+resource "tls_private_key" "private_keys" {
```

Comment:
We should move the cert creation to the example, so providing them isn't required.

Reply:
Done: https://github.com/terraform-google-modules/terraform-google-pubsub/pull/4/commits/ad220b7e8b908ba11664e60e27aa519d1d89fb73